20xt/src/baja_ipc/include/ipc_config.h

```diff
@@ -0,0 +1,68 @@
+#ifndef STATION_IPC_CONFIG_H
+#define STATION_IPC_CONFIG_H
+
+#include <vector>
+#include <string>
+#include <mqueue.h>
+
+namespace BajaIPC {
+  const uint32_t MAX_QUEUE_MSG_SIZE = 1024;
+  const uint32_t MAX_RADIO_MSG_SIZE = 100;
+
+  const int QUEUE_MODE = 0644;
+  const int QUEUE_FLAGS = O_CREAT | O_RDWR;
+  const mq_attr QUEUE_ATTRIBUTES = {
+    .mq_flags = 0,
+    .mq_maxmsg = 10,
+    .mq_msgsize = MAX_QUEUE_MSG_SIZE,
+    .mq_curmsgs = 0,
+  };
+
+  // Return codes detailed in POSIX message queues
+  const mqd_t open_queue(std::string q_fname, bool blocking);
+  const int close_queue(mqd_t qid);
+  const int unlink_queue(std::string q_fname);
+
+  // Get a message from the queue, return empty string if no message. Non blocking
+  const std::string get_message(mqd_t qid);
+  const int num_messages_available(mqd_t qid);
+
+  // Send a message to the queue
+  enum SendResult {
+    SUCCESS,
+    QUEUE_FULL,
+    SEND_ERROR,
+  };
+  const int send_message(mqd_t qid, const std::string& msg);
+}
+
+namespace StationIPC {
+  // Queues of LiveComm from the car or bound to the car
+  const std::string XBEE_DRIVER_TO_TX_QUEUE = "/xbee_driver_tx_queue";
```

Comment:
All of the StationIPC queues can be renamed to follow the pattern in CarIPC. I think those names are more readable. We only need to change this file.